samples/matter/common/src/bridge/bridge_storage_manager.h

```diff
@@ -7,7 +7,7 @@
 #pragma once
 
 #include "matter_bridged_device.h"
-#include "persistent_storage_util.h"
+#include "ps_storage/persistent_storage_util.h"
```

Comment:
Regarding the commit description, I can see the commit doesn't implement anything new. I guess, you've used the previously created common binding manager/handler in the Matter Bridge, right? :)